randovania/games/prime1/layout/preset_describer.py

```diff
@@ -44,6 +44,31 @@ def describe_probability(probability, attribute):
         superheated_probability = describe_probability(configuration.superheated_probability, "superheated")
         submerged_probability = describe_probability(configuration.submerged_probability, "submerged")
 
+        def attribute_in_range(rand_range, attribute):
+            if rand_range[0] == 1.0 and rand_range[1] == 1.0:
+                return None
+            elif rand_range[0] > rand_range[1]:
+                temp = rand_range[0]
+                rand_range[0] = rand_range[1]
+                rand_range[1] = temp
+
+            return "randomized " + attribute + " within range " + str(rand_range[0]) + " - " + str(rand_range[1])
+
+        def different_xyz_randomization(diff_xyz):
+            if enemy_rando_range_scale is None:
+                return None
+            elif diff_xyz == True:
+                return "Scale XYZ values will be randomized separately"
```

Comment:
Thinking about this, calling this case as just "stretching" (or something similar) works great.
Talking about changing x/y/z separately is not intuitive if you're not familiar to how models can be scaled.